schema.txt

```diff
@@ -0,0 +1,19 @@
+{
```

Comment:
Why you didn't create a folder call tests and move it there ?

Reply:
I did but did not push it to GitHub
and did not put this file in it because it is an essential file 
but after mentioning it and thinking again you are right and I will.